File: 백준/Bronze/1085. 직사각형에서 탈출/직사각형에서 탈출.c
#include <stdio.h>

int main()
{
    int x, y, w, h;
    scanf("%d %d %d %d", &x, &y, &w, &h);
    int answer = x;
    
    if (answer > w - x) answer = w - x;
    if (answer > y) answer = y;
    if (answer > h - y) answer = h - y;
    
    printf("%d", answer);
}

File: 백준/Bronze/1085. 직사각형에서 탈출/직사각형에서 탈출.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    
	    int x = sc.nextInt(), y = sc.nextInt(), w = sc.nextInt(), h = sc.nextInt(), answer = x;
        
        if (answer > w - x) answer = w - x;
        if (answer > y) answer = y;
        if (answer > h - y) answer = h - y;
	    
		System.out.println(answer);
	}
}